src/dune/global.mli

```diff
@@ -0,0 +1,3 @@
+open Stdune
+
+val env : Env.t Fdecl.t
```

Comment:
Why not `Env.global`? I'm just thinking that we have other globals in Dune, and introducing a `Global` module might give the fasle impression that this module contains all the globals of Dune.

Reply:
Because Env belongs to stdune. I’d prefer to not have dune specific in it if possible.

Reply:
Make sense